src/shared-components/button/index.js

```diff
@@ -56,6 +58,7 @@ class Button extends React.Component {
       isLoading,
       icon,
       textColor,
+      fullWidth,
```

Comment:
I opted for `fullWidth` over `isFullWidth` because I anticipated it would be a value we would nearly always be setting explicitly, rather than conditionally.